pageObjects/NotesPage.js

```diff
@@ -17,12 +18,19 @@ class NotesPage extends BasePage {
         this.newNoteTitleField.click()
         this.newNoteTitleField.sendKeys(title)
         element(by.buttonText('Save')).click()
-        browser.sleep(browser.params.customTimeout)
+        if (title !== '' && body !== '') {
```

Comment:
It won't help here, as we need to validate if `title !== '' && body !== ''` anyway to execute or not execute the waiting:
```
browser.wait(EC.presenceOf(this.noteFirst), browser.params.customTimeout,
                'Success notification should be visible after note creation')
```
This wait is not needed if `title === '' && body === ''` because the note was not created